genesis/engine/bvh.py

```diff
@@ -177,6 +183,16 @@ def build(self):
         self.build_radix_tree()
         self.compute_bounds()
 
+    @ti.func
+    def filter(self, i_a, i_q):
+        """
+        Default filter function that always returns False.
+        This function can be overridden by registering a custom filter.
```

Comment:
You should mention that it has a default implementation and explain what the default does.